templates/mstest.tmpl

```diff
@@ -0,0 +1,12 @@
+${namespaces}namespace ${namespace}
```

Comment:
I assume this will be the injected  somehow based on the test location, right?

Reply:
Do you mean the `using XUnit` etc (${namespaces} template var}? That is covered by the array part in these lines: https://github.com/KreativJos/csharpextensions/blob/77eeffed180ba780ca85a7e23187f5d35008781a/src/extension.ts#L155-L157

The ${namespace} is handled like all other templates, via `NamespaceDetector.getNamespace` - https://github.com/KreativJos/csharpextensions/blob/77eeffed180ba780ca85a7e23187f5d35008781a/src/namespaceDetector.ts#L16